scripts/installer/glassfish-setup.sh

```diff
@@ -157,121 +157,128 @@ if [  $(echo $DOMAIN_DOWN|wc -c) -ne 1  ];
 
-# http://docs.oracle.com/cd/E18930_01/html/821-2416/abllx.html#giogt
-./asadmin $ASADMIN_OPTS create-connector-resource --poolname jms/IngestQueueConnectionFactoryPool --description "ingest connector resource" jms/IngestQueueConnectionFactory
+  # created mail configuration: 
 
-# http://docs.oracle.com/cd/E18930_01/html/821-2416/ablmc.html#giolr
-./asadmin $ASADMIN_OPTS create-admin-object --restype javax.jms.Queue --raname jmsra --description "sample administered object" --property Name=DataverseIngest jms/DataverseIngest
+  ./asadmin $ASADMIN_OPTS create-javamail-resource --mailhost "$SMTP_SERVER" --mailuser "dataversenotify" --fromaddress "do-not-reply@${HOST_ADDRESS}" mail/notifyMailSession
+
+  # so we can front with apache httpd ( ProxyPass / ajp://localhost:8009/ )
+  ./asadmin $ASADMIN_OPTS create-network-listener --protocol http-listener-1 --listenerport 8009 --jkenabled true jk-connector
+ else
+    echo $MY_POD_NAME
+    if [ $MY_POD_NAME == "dataverse-glassfish-0" ]
+      then
+        echo "I am in a container so I am doing much less"
+        ./asadmin $ASADMIN_OPTS delete-jvm-options -Xmx512m
+        ./asadmin $ASADMIN_OPTS create-jvm-options "-Xmx${MEM_HEAP_SIZE}m"
 
-# no need to explicitly create the resource reference for the connection factory created above -
-# the "create-connector-resource" creates the reference automatically.
-#./asadmin $ASADMIN_OPTS create-resource-ref --target Cluster1 jms/IngestQueueConnectionFactory
 
-# created mail configuration: 
+        ./asadmin $ASADMIN_OPTS create-jdbc-connection-pool --restype javax.sql.DataSource \
+                                        --datasourceclassname org.postgresql.ds.PGPoolingDataSource \
+                                        --property create=true:User=$DB_USER:PortNumber=$DB_PORT:databaseName=$DB_NAME:password=$DB_PASS:ServerName=$DB_HOST \
+                                        dvnDbPool
 
-./asadmin $ASADMIN_OPTS create-javamail-resource --mailhost "$SMTP_SERVER" --mailuser "dataversenotify" --fromaddress "do-not-reply@${HOST_ADDRESS}" mail/notifyMailSession
+        ###
+        # Create data sources
+        ./asadmin $ASADMIN_OPTS create-jdbc-resource --connectionpoolid dvnDbPool jdbc/VDCNetDS
 
-# so we can front with apache httpd ( ProxyPass / ajp://localhost:8009/ )
-./asadmin $ASADMIN_OPTS create-network-listener --protocol http-listener-1 --listenerport 8009 --jkenabled true jk-connector
+        ###
+        # Set up the data source for the timers
+
+        ./asadmin $ASADMIN_OPTS set configs.config.server-config.ejb-container.ejb-timer-service.timer-datasource=jdbc/VDCNetDS
+
+        ./asadmin $ASADMIN_OPTS create-jvm-options "\-Djavax.xml.parsers.SAXParserFactory=com.sun.org.apache.xerces.internal.jaxp.SAXParserFactoryImpl"
+
+        ./asadmin $ASADMIN_OPTS create-javamail-resource --mailhost "$SMTP_SERVER" --mailuser "dataversenotify" --fromaddress "do-not-reply@${HOST_ADDRESS}" mail/notifyMailSession
+
+      fi
+  fi
```

Comment:
I'm confused by all the whitespace changes above. What was actually changed in this script?

Reply:
If you notice, the change isn't the white space. It's the different procedures being called if MY_POD_NAME exists!